app/components/concept-page/content.ts

```diff
@@ -33,6 +33,13 @@ export default class ContentComponent extends Component<Signature> {
     return this.args.allConcepts.find((concept) => concept.slug === this.args.conceptGroup?.nextConceptSlug(this.args.concept.slug));
   }
 
+  get remainingBlocksCount() {
```

Comment:
This method will fit well in `ConceptEngagement`. That way you could pass in `@conceptEngagement` directly to `ConceptProgress`, and it could use both `engagement.currentProgressPercentage` and `remainingBlocksCount`